service_notification/src/main/kotlin/org/thechance/service_notification/domain/usecases/UserUseCase.kt

```diff
@@ -0,0 +1,20 @@
+package org.thechance.service_notification.domain.usecases
+
+import org.koin.core.annotation.Single
+import org.thechance.service_notification.domain.gateway.UserGateway
+import org.thechance.service_notification.domain.model.User
+
+@Single
+class UserUseCase(private val userGateway: UserGateway): IUserUseCase {
```

Comment:
What about user management use case?

Reply:
what's its use? @mosayed01

Reply:
It's story to manage users notifications